tests/qemu-tests/test_qemu.sh

```diff
@@ -2,13 +2,59 @@
 
 make -C binaries
 
-qemu-aarch64 \
-    -g 1234 \
-    -L /usr/aarch64-linux-gnu/ \
-    ./binaries/reference-binary.aarch64.out &
-
-gdb-multiarch \
-    -ex "file ./binaries/reference-binary.aarch64.out" \
```

Comment:
Are we effectively not having any test that use reference-binary.aarch64.out right now?